test_programs/plonky2_trace/3_two_files/expected_trace.json

```diff
@@ -0,0 +1,19 @@
+[
+  { "Path": "?" },
+  { "Function": { "path_id": 0, "line": -1, "name": "<toplevel>" } },
+  { "Call": { "function_id": 0, "args": [] } },
+  { "Type": { "kind": 30, "lang_type": "None", "specific_info": { "kind": "None" } } },
+  { "Path": "/home/stan/code/repos/noir/test_programs/plonky2_trace/3_two_files/src/main.nr" },
+  { "Step": { "path_id": 1, "line": 6 } },
```

Comment:
It is nice that `path_id` works fine, but do you think it is possible to add some mapping between the id i and the file it represents. (`Id <> relative path` for example )
My point is, if there are more files it would be really hard to keep track. Or maybe I should not care for that?

Reply:
We do not control the format, so we cannot add new fields. But you're right.

Leaving as is.